web/template/watch.gohtml

```diff
@@ -305,13 +311,6 @@
                     {{/* Lecture hall is set, means no self-stream*/}}
                     {{if $stream.LiveNow}}
                         {{if $stream.LectureHallID}}
-                            <button @click="fetch(`/api/stream/${streamID}/pause?pause=${!paused}`).then(paused=!paused)"
```

Comment:
This button shouldn't be there right? As far as I know there isn't a pause function anymore 😅

Reply:
Yes :+1: